cloud/ec2.go

```diff
@@ -1187,12 +1187,7 @@ func (m *ec2Manager) CostForDuration(ctx context.Context, h *host.Host, start, e
 	if end.Before(start) || util.IsZeroTime(start) || util.IsZeroTime(end) {
 		return 0, errors.New("task timing data is malformed")
 	}
-	ec2Settings := &EC2ProviderSettings{}
-	err := ec2Settings.fromDistroSettings(h.Distro)
-	if err != nil {
-		return 0, errors.Wrap(err, "problem getting region from host")
-	}
-	if err = m.client.Create(m.credentials, m.region); err != nil {
+	if err := m.client.Create(m.credentials, m.region); err != nil {
```

Comment:
Why remove `fromDistroSettings` here?

Reply:
it's not strictly related to the PR, but it's unused.